runtime/ms-rest-azure/lib/credentials/msiTokenCredentials.js

```diff
@@ -12,25 +12,25 @@ class MSITokenCredentials {
       throw new Error('domain must be a non empty string.');

```

Comment:
Please take a look at the flow implemented for MSI auth flow in CLI. 
- https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/azure/cli/core/_profile.py#L532-L592.

The CLI asks for the tenantId against which the token is required. We are not asking that from the customer.